<launch>
  <group ns="laser">
    <node pkg="nodelet" type="nodelet" name="imu_manager" args="manager" output="screen" />

    <node pkg="nodelet" type="nodelet" name="PhidgetsImuNodelet" args="load phidgets_imu/PhidgetsImuNodelet imu_manager" output="screen" >
      # supported data rates: 4 8 16 24 32 40 ... 1000 (in ms)
      <param name="period" value="4" />
      <param name="frame_id" value="imu_laser" />
    </node>

    <node pkg="nodelet" type="nodelet" name="ImuFilterNodelet" args="load imu_filter_madgwick/ImuFilterNodelet imu_manager" output="screen" >
      <param name="use_mag" value="true" />
      <param name="publish_tf" value="false" />
    </node>

    <node pkg="scout3d_laser" type="lasercontroller_node.py" name="lasercontroller_node"/>

    <node pkg="scout3d_temperature" type="temperature_node.py" name="temperature_node"/>
  </group>
</launch>
